MapboxDirections/MBComponent.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+/**
+The component protocol that comprises what the instruction banner should display.
+ */
+@objc(MBComponent)
+public protocol Component: class, NSSecureCoding {
```

Comment:
Component is a pretty generic term, and a protocol’s name should describe some quality about its conforming classes. Since the two conforming classes are VisualInstructionComponent and LaneIndicationComponent, both of which are intended for visual display, let’s call the protocol something like Visualizable. (This isn’t a great name – any better ideas?)